packages/web/src/components/Button/_Button.scss

```diff
@@ -24,7 +24,8 @@
         text-decoration: none;
     }
 
-    &:disabled {
+    &:disabled,
+    &.Button--disabled {
```

Comment:
@adamkudrna Should be a prefix also applicable here?

Reply:
I don't know how to test it

Reply:
> @adamkudrna Should be a prefix also applicable here?

Yes please!

Reply:
Prefix yes, but—is the _class_ itself necessary? Why is `:disabled` not sufficient?

Reply:
its necessary for anchor tag, because :disabled is not working there

Reply:
I have answer to my question. `Button--disabled` will by also prefixed with PostCSS when using prefix like `jobs`:

```css
.jobs-Button:disabled, .jobs-Button.jobs-Button--disabled {
  cursor: default;
  pointer-events: none;
}
```